promgen/models.py

```diff
@@ -210,6 +210,7 @@ class Service(models.Model):
     )
 
     notifiers = GenericRelation(Sender)
+    project_set = GenericRelation("Project")
```

Comment:
```suggestion
```

I believe this is not actually needed. This is already defined by Django automatically due to the Project model having a foreign key to Service

https://github.com/line/promgen/blob/0dc687f1487ed627f149bc45c060fb3f20162b48/promgen/models.py#L244

https://docs.djangoproject.com/en/3.2/ref/models/fields/#django.db.models.ForeignKey.related_name

GenericRelation is probably incorrect since GenericRelation is supposed to be used with GenericForeignKey which is why it is used with Senders and Rules

https://github.com/line/promgen/blob/0dc687f1487ed627f149bc45c060fb3f20162b48/promgen/models.py#L416
https://github.com/line/promgen/blob/0dc687f1487ed627f149bc45c060fb3f20162b48/promgen/models.py#L82

Reply:
True 👍🏻 I have removed that part.